### PR TITLE
Document example for running download as pod

### PR DESCRIPTION
**Pull request by donpenney**
/cc @browsell @imiller0

### CODE REVIEW

File: docs/reinstall/090-config-map.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            - name: sriov-network-operator
+              channels:
+                - name: 'stable'
+            - name: cluster-logging
+              channels:
+                - name: 'stable'
+            - name: lvms-operator
+              channels:
+                - name: 'stable-4.12'
+            - name: amq7-interconnect-operator
```
Comment (browsell) [inline docs/reinstall/090-config-map.yaml:67]: Remove

File: docs/reinstall/090-config-map.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+              channels:
+                - name: '1.10.x'
+            - name: bare-metal-event-relay
```
Comment (browsell) [inline docs/reinstall/090-config-map.yaml:70]: Remove

File: docs/reinstall/090-config-map.yaml (hunk 1 above)
Comment (donpenney) [reply docs/reinstall/090-config-map.yaml]: Done

File: docs/reinstall/090-config-map.yaml (hunk 2 above)
Comment (donpenney) [reply docs/reinstall/090-config-map.yaml]: Done

File: docs/reinstall/090-config-map.yaml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+      - thanos
+      - vsphere
+  imageset.yaml: |
+    ---
+    apiVersion: mirror.openshift.io/v1alpha2
+    kind: ImageSetConfiguration
+    mirror:
+      platform:
+        channels:
+        - name: stable-4.12
```
[the remark this answers is not in the document]
Comment (donpenney) [reply docs/reinstall/090-config-map.yaml:51]: Added comment blocks to the yaml to describe the purpose of the examples, and, in the case of the imageset.yaml, how it was generated

**Pull request merged**